infrastructure/api/resources/vendors.py

```diff
@@ -0,0 +1,20 @@
+from infrastructure.api.resources.base import BaseResource
+
+
+# TODO: request & response models (with marshmallow for example)
```

Comment:
the vendor resource implementations are just simple examples without implementation, to verify that the whole thing is working :D But we have to implement them anyway later